.gitignore

```diff
@@ -57,3 +57,8 @@ typings/
 # dotenv environment variables file
 .env
 
+# system files
+.DS_Store
+
+# project files
+index.js
```

Comment:
Why `index.js`?

Reply:
Is it the output of a build step?

Reply:
I see here: https://github.com/bucko13/bsock-middleware/blob/master/package.json#L8